Sources/NIO/EventLoop.swift

```diff
@@ -733,6 +766,11 @@ public protocol EventLoopGroup: class {
     /// The virtue of this function is to shut the event loop down. To work around that we call back on a DispatchQueue
     /// instead.
     func shutdownGracefully(queue: DispatchQueue, _ callback: @escaping (Error?) -> Void)
+
+    /// Returns an `EventLoopIterator` over the `EventLoop`s in this `EventLoopGroup`.
+    ///
+    /// - returns: `EventLoopIterator`
```

Comment:
we should add something like

    /// - note: The return value of `makeIterator` is currently optional as requiring it would be SemVer major. From NIO 2.0.0 on it will return a non-optional iterator.

and please file an issue (or separate PR (that will only be merged when 2.0.0 gets released)) to do so.